libs/examples/src/indicator/indicator-examples.module.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { NgModule } from '@angular/core';
+import { DtIndicatorModule } from '@dynatrace/barista-components/indicator';
+
+import { DtExampleIndicatorDefault } from './indicator-default-example/indicator-default-example';
+import { DtTreeTableModule } from '@dynatrace/barista-components/tree-table';
+import { DtIconModule } from '@dynatrace/barista-components/icon';
+import { DtContextDialogModule } from '@dynatrace/barista-components/context-dialog';
+import { DtInfoGroupModule } from '@dynatrace/barista-components/info-group';
+import { DtButtonModule } from '@dynatrace/barista-components/button';
+import { DtProgressBarModule } from '@dynatrace/barista-components/progress-bar';
+
+export const DT_INFO_GROUP_EXAMPLES = [DtExampleIndicatorDefault];
+
+@NgModule({
+  imports: [
+    DtIndicatorModule,
+    DtTreeTableModule,
+    DtIconModule,
+    DtContextDialogModule,
+    DtInfoGroupModule,
+    DtButtonModule,
+    DtProgressBarModule,
```

Comment:
```suggestion
    DtButtonModule,
```
Button and Indicator module should be enough for a simple example